<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="get_url_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QLabel" name="endpoint_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ENDPOINT: </string>
    </property>
   </widget>
   <widget class="QLabel" name="btc_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>BTC</string>
    </property>
   </widget>
   <widget class="QLabel" name="eth_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ETH</string>
    </property>
   </widget>
   <widget class="QLabel" name="ada_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ADA</string>
    </property>
   </widget>
   <widget class="QLabel" name="btc_value_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="eth_value_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="ada_value_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="endpoint_value_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>https://api.exchangerate.host/latest?source=crypto</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="base_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Base:</string>
    </property>
   </widget>
   <widget class="QLabel" name="last_fetched_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Fetched:</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_value_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="base_value_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="last_fetched_value_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
